lib/rest/RequestHandler.js

```diff
@@ -267,15 +267,7 @@ class RequestHandler {
 
                         this.ratelimits[route].remaining = resp.headers["x-ratelimit-remaining"] === undefined ? 1 : +resp.headers["x-ratelimit-remaining"] || 0;
 
-                        let retryAfter = parseInt(resp.headers["retry-after"]);
-                        // Discord breaks RFC here, using milliseconds instead of seconds (╯°□°）╯︵ ┻━┻
-                        // This is the unofficial Discord dev-recommended way of detecting that
-                        if(retryAfter && (typeof resp.headers["via"] !== "string" || !resp.headers["via"].includes("1.1 google"))) {
-                            retryAfter *= 1000;
-                            if(retryAfter >= 1000 * 1000) {
-                                this._client.emit("warn", `Excessive Retry-After interval detected (Retry-After: ${resp.headers["retry-after"]} * 1000, Via: ${resp.headers["via"]})`);
-                            }
-                        }
+                        const retryAfter = parseInt(resp.headers["x-ratelimit-reset-after"] || resp.headers["retry-after"]) * 1000;
```

Comment:
~~x-ratelimit-reset-after is always returned in ms precision in v8/v9, only retry-after needs to be * 1000~~

nvm its returned as a decimal not a full ms value, nevermind me